nemo_skills/mcp/servers/wikipedia_tool.py

```
@@ -0,0 +1,116 @@
+            return "No Wikipedia articles found for this query."
+
+        results = []
+        for title in titles:
+            try:
+                page = wikipedia.page(title, auto_suggest=False)
+                summary = page.summary[:300] + ("..." if len(page.summary) > 300 else "")
+                results.append(f"**{page.title}**\nURL: {page.url}\n{summary}")
+            except (wikipedia.DisambiguationError, wikipedia.PageError):
+                results.append(f"**{title}** (disambiguation or not found)")
+        return "\n\n---\n\n".join(results)
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)